recipe/meta.yaml

```diff
@@ -21,23 +21,24 @@ requirements:
     - hatchling >=1.4.0
     - hatch-nodejs-version
     - hatch-jupyter-builder
-    - jupyterlab >=4.0.0
+    - jupyterlab {{ jupyterlab_version }}
   run:
     - python >=3.8
     - jupyter_server >=2.0.0,<3.0.0
-    - jupyter_ydoc >=1.0.1,<2.0.0
+    - jupyter_ydoc >=1.1.0a0,<2.0.0
```

Comment:
I don't think this needs to be updated. Could you please confirm this @hbcarlos?

Reply:
No, we don't need to update it.

Reply:
resynced with the correct tag (instead of `main`), appears fine now.